src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -168,15 +168,15 @@ export interface IResultsViewPageProps {
 }
 
 export enum ResultsViewTab {
-    ONCOPRINT="oncoprintTab",
-    CANCER_TYPES_SUMMARY="cancerTypesSummaryTab",
-    MUTUAL_EXCLUSIVITY="mutualExclusivityTab",
+    ONCOPRINT="oncoprint",
+    CANCER_TYPES_SUMMARY="cancerTypesSummary",
+    MUTUAL_EXCLUSIVITY="mutualExclusivity",
     PLOTS="plots",
-    MUTATIONS="mutationsTab",
+    MUTATIONS="mutations",
     COEXPRESSION="coexpression",
-    ENRICHMENT="enrichment",
-    SURVIVAL="survivalTab",
-    CN_SEGMENTS="copyNumberSegmentsTab",
+    ENRICHMENT="enrichments",
+    SURVIVAL="survival",
+    CN_SEGMENTS="cnSegments",
```

Comment:
you should do a grep to make sure those names werent used elsewhere, because I didn't do that when I added this enum

Reply:
oh great!